cfme/tests/automate/test_instance.py

```diff
@@ -219,3 +219,63 @@ def test_automate_relationship_trailing_spaces(request, klass, namespace, domain
             assert result.output == ""
         else:
             assert search in result.output
+
+
+@pytest.fixture(scope="module")
+def copy_instance(domain):
+    """
+    This fixture copies the instance '/ManageIQ/System/Request/ansible_tower_job' to new domain.
+    """
+    # Instantiating class 'Request'
+    klass = (
+        domain.parent.instantiate(name="ManageIQ")
+        .namespaces.instantiate(name="System")
+        .classes.instantiate(name="Request")
+    )
+
+    # Instantiating instance 'ansible_tower_job' and copying it to new domain
+    klass.instances.instantiate(name="ansible_tower_job").copy_to(domain.name)
+    instance = (
+        domain.namespaces.instantiate(name="System")
+        .classes.instantiate(name="Request")
+        .instances.instantiate(name="ansible_tower_job")
+    )
+    yield instance
+
+
+@pytest.mark.tier(1)
+def test_check_system_request_calls_depr_conf_mgmt(appliance, copy_instance):
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseimportance: low
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.10
+        casecomponent: Automate
+        tags: automate
+        setup:
+            1. Copy /System/Request/ansible_tower_job instance to new domain
+        testSteps:
+            1. Run that instance(ansible_tower_job) using simulation
+            2. See automation log
+        expectedResults:
+            1.
+            2. The /System/Request/ansible_tower_job instance should call the newer
+               "/AutomationManagement/AnsibleTower/Operations/StateMachines/Job/default method"
+
+    Bugzilla:
+        1615444
+    """
+    search = '/AutomationManagement/AnsibleTower/Operations/StateMachines/Job/default'
+
+    # Executing the automate instance - 'ansible_tower_job' using simulation
+    simulate(
+        appliance=appliance,
+        request=copy_instance.name
+    )
+    result = appliance.ssh_client.run_command(
```

Comment:
Have you considered using `LogValidator` here instead? https://github.com/ManageIQ/integration_tests/blob/master/cfme/utils/log_validator.py

That way you can verify exactly that the string is found in the log. 

There are a many examples throughout the code.